tribe-ext-settings-import-export.php

```diff
@@ -208,43 +207,24 @@ public function tribe_sie_settings_page() {
 		/**
 		 * Process a settings export that generates a .json file of the shop settings
 		 */
-		function tribe_sie_process_settings_export() {
-			// Bail if no action
-			if ( empty( $_POST['tribe_sie_action'] ) || 'export_settings' != $_POST['tribe_sie_action'] ) {
-				return;
-			}
-			// Bail if no nonce
-			if ( ! wp_verify_nonce( $_POST['tribe_sie_export_nonce'], 'tribe_sie_export_nonce' ) ) {
-				return;
-			}
-			// Bail if no capability
-			if ( ! current_user_can( 'manage_options' ) ) {
-				return;
-			}
-
-			$settings = get_option( 'tribe_events_calendar_options' );
-
-			ignore_user_abort( true );
-			nocache_headers();
-			header( 'Content-Type: application/json; charset=utf-8' );
-			header( 'Content-Disposition: attachment; filename=tribe-settings-export-' . date( 'm-d-Y' ) . '.json' );
-			header( "Expires: 0" );
-			echo json_encode( $settings );
-			exit;
-		}
-
-		/**
-		 * Process a settings import from a json file
-		 */
-		function tribe_sie_process_settings_import() {
-
-			// Bail if no action
-			if ( empty( $_POST['tribe_sie_action'] ) || 'import_settings' != $_POST['tribe_sie_action'] ) {
+		function tribe_sie_process_settings_action() {
```

Comment:
This is just a note/observation (I wanted to share the thought but it's definitely not essential you change it or anything): prefixing methods with `tribe_sie_` probably isn't needed, because we're already inside the `Tribe\Extensions\Settings_Import_Export` namespace (if this were a top-level function in the global namespace, the prefixing would definitely be a good thing...but since we're not we could probably stick to a more concise naming convention that focuses on what the method does and not what component it belongs to).

Reply:
Great advice. "I knew that" - I could say. :) It's just not in muscle memory yet. :) Resolved.